README.md

```diff
@@ -211,64 +262,68 @@ Alternatively, refactor the code to use `<img>` tags.
 
 1. Add a `<title>` node inside of the svg that describes the image.
-  
 
 ```html
 <svg>
   <title>description of svg</title>
 </svg>
 ```
 
-1. Hide the svg via `aria-hidden="true"` and add a visually hidden element that describes it.
+2. Hide the svg via `aria-hidden="true"` and add a visually hidden element that describes it.
 ```html
-<span class='visually-hidden'>description of svg</span>
-<svg aria-hidden='true'></svg>
+<span class="visually-hidden">description of svg</span>
+<svg aria-hidden="true"></svg>
 ```
 
 
-# Input Labels
+## <a name="inputLabels"></a>Input Labels
+
 Each input should have a `<label>` associated with it:
 ```html
 <label for="firstname">First name:</label>
 <input type="text" name="firstname" id="firstname">
 ```
 
-Sometimes we don't want to visually display the `<label>` so we can **visually hide** the element using our previously described `visually-hidden` class:
+Sometimes we don't want to visually display the `<label>` so we can [**visually hide**](#hiding) the element using our previously described `visually-hidden` class:
 
 ```html
 <label for="firstname" class="visually-hidden">First name:</label>
 <input type="text" name="firstname" id="firstname">
 ```
 
 
-# Input Validation
+## <a name="inputValidation"></a>Input Validation
 
-Input are often given different styling to denote invalid input. This styling may not be obvious to assistive technologies.
-Instructing screen readers that a specific input is invalid can be done via the [aria-invalid](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute) attribute.
+Inputs are often given different styling to denote an invalid input. This styling may not be obvious to assistive technologies. Styling should never be the sole way of denoting an invalid input. Instead styling should be used in conjunction with a text message and the [aria-invalid](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute) attribute.
 
 ```html
-<input aria-invalid='true' />
+<input aria-invalid="true" />
 ```
 
-There are exceptions where the inputs are styled as visually disabled and they don't have the `disabled` attribute declared.
+## <a name="disablingInputs"></a>Disabling Inputs
+
 Assistive technologies wouldn't announce the disabled styling so the disabled state should be indicated using the [aria-disabled](https://www.digitala11y.com/aria-disabled-state/) attribute. 
 
-For example: a button needs to be shown as disabled and it should still be clickable.
-Maybe clicking on the disabled button triggers a validation announcement.
+There are exceptions where the inputs are styled as visually disabled and they don't have the `disabled` attribute declared. For example: a checkbox needs to be visually disabled but it should still be focusable. We'd want this so its content is still accessible to the screen reader.
```

Comment:
Not sure this checkbox example is an actual use-case. We wouldn't denote a checkbox as disabled but still have it checkable. It would be conflicting information to the user.
We did use the pattern where buttons are disabled but still clickable and when you do that it triggers the form validation

Reply:
@aoxrud-ww I was confused here. I thought aria-disabled hid elements from the screen reader but that's not the case. That being said I think this section could benefit from a brief discussion of `disabled` vs `aria-disabled` 
```suggestion
For example: a button needs to be shown as disabled and it should still be clickable.
Maybe clicking on the disabled button triggers a validation announcement.
It's important to note that the `disabled` attribute will hide the element from the screen reader. This should be avoided because although the element is disabled it still provides valuable information to the user. For example if a next button has a `disabled` (instead of an `aria-disabled`) attribute by default until some condition is met that button will be hidden from the screen reader. Knowing that there **is** a next button and it's disabled is valuable information which can be conveyed through `aria-disabled`.
```